include/ros2_logging_fmt/ros2_logging_fmt.hpp

```diff
@@ -15,49 +15,113 @@ class Logger
   Logger(rclcpp::Logger rclcpp_logger): rclcpp_logger_(rclcpp_logger)
   {}
 
+  Logger(rclcpp::Logger rclcpp_logger, rclcpp::Clock::SharedPtr clock): rclcpp_logger_(rclcpp_logger)
+  { clk = clock; }
+
   template<typename... Args>
   void info(const char* str, Args... args)
   {
     RCLCPP_INFO(rclcpp_logger_, write_buffer(str, args...) );
   }
 
+  template<typename... Args>
+  void info_throttle(const char* str, rclcpp::Clock::SharedPtr clock, int duration, Args... args)
+  {
+      RCLCPP_INFO_THROTTLE(rclcpp_logger_, *clock, duration, write_buffer(str, args...) );
+  }
+
+  template<typename... Args>
+  void info_throttle(int duration, const char* str, Args... args)
```

Comment:
Shouldn't you always place the order of args:

[clock], duration, str, args...

In the version with the clock arg you ordered them differently.

Reply:
Gah, this was an overlook on my part as I was working on the code.